content/influxdb/cloud/organizations/users.md

```diff
@@ -24,7 +24,10 @@ By default, each user has full permissions on resources in your organization.
   - [Resend an invitation](#resend-an-invitation)
   - [Withdraw an invitation](#withdraw-an-invitation)
 - [Remove a user from your organization](#remove-a-user-from-your-organization)
-  - [Remove yourself from an organization](#remove-yourself-from-an-organization)
+
+{{% note %}}
+**Tip:** If you have multiple users, we recommend creating a system account for administrative purposes. For example, if you have applications writing to InfluxDB, ensure the applications use the system account credentials.
```

Comment:
we generally use NOTE instead of TIP

Reply:
thank you, @lwandzura -- I told @peterreg I thought tip was more descriptive :-) but you're right, we typically use note